src/main/java/com/revature/repositories/ResultSummaryCsvRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.revature.repositories;
+
+import com.revature.models.ResultSummaryCsv;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ResultSummaryCsvRepository extends JpaRepository<ResultSummaryCsv, Integer> {
+    
+    ResultSummaryCsv findById(int id);
```

Comment:
We found this to be more convenient than the built in findById

Reply:
Is that due the `Optional` wrapper? In general, I think `Optional` is preferred over just the raw type. Since no matter what, there is a possibility of the value being non-existent. The `Optional` wrapper forces the developer to handle that edge case. Otherwise, it could be forgotten, leading to `NullPointerException`.

Reply:
Code was refactored to use Optional instead.